src/main/app/src/components/toteutus/Toteutus.js

```diff
@@ -15,7 +15,7 @@ import { Localizer as l } from '#/src/tools/Utils';
 import HakuKaynnissaCard from '#/src/components/koulutus/HakuKaynnissaCard';
 import TeemakuvaImage from '#/src/components/common/TeemakuvaImage';
 import OpenInNewIcon from '@material-ui/icons/OpenInNew';
-import _ from 'lodash';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
mielestäni luettavuus parempi, jos importoidaan koko `lodash`in alaviivan kautta ja siinä missä käytetään on `_.isEmpty(...)`